src/states/approved_peer/mod.rs

```diff
@@ -1412,13 +1405,13 @@ impl ApprovedPeer {
     }
 
     fn vote_for_relocate(&mut self, details: RelocateDetails) {
-        self.vote_for_network_event(details.into_accumulating_event().into_network_event())
+        self.parsec_map
+            .vote_for(details.into_accumulating_event().into_network_event())
     }
 
     fn vote_for_relocate_prepare(&mut self, details: RelocateDetails, count_down: i32) {
-        self.vote_for_network_event(
-            AccumulatingEvent::RelocatePrepare(details, count_down).into_network_event(),
-        );
+        self.parsec_map
+            .vote_for(AccumulatingEvent::RelocatePrepare(details, count_down).into_network_event());
```

Comment:
As with many other comments, not part of this PR, but taking the chance to comment on it:

`RelocatePrepare` has an odd name. An event is in past tense, is it meant to be `RelocationPrepared`? Or is this a vote for a _cmd_ of `PrepareRelocation`?

Reply:
Yeah, `PrepareRelocation` would be more appropriate I think. Also, despite of it being called `AccumulatingEvent`, it's not really an event in the traditional sense. It's the thing we input into Parsec and expect to come out of it in consistent order. Perhaps we should come up with different name for it?

Reply:
Aha OK, good. I think I have just the thing for it then. Will think a bit and possibly include it in the POC or in the docs for it.